remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
+class LivySession(GuidMixin, object):
```

Comment:
Here is the code, I tried:

class GuidMixin(object):
    def __init__(self):
        self.guid = generate_uuid()

class Subclass(GuidMixin):

    def printx(self):
        print self.guid

if __name__ == "__main__":
    a = Subclass()
    b = Subclass()

    print a.guid
    a.printx()

    print b.guid
    b.printx()